mtools/util/logevent.py

```diff
@@ -638,7 +650,9 @@ def _extract_counters(self):
             'nDeleted': 'ndeleted',
             'nInserted': 'ninserted',
             'nMatched': 'nreturned',
-            'nModified': 'nupdated'
+            'nModified': 'nupdated',
+            'cursorid' : 'cursorid',
+            'repaedtime' : 'reapedtime'
```

Comment:
there is a type **repaedtime** and **reapedtime**, how do use this key value pairs?

Reply:
Most probably typos for the same token (variable). Will fix that.